lib/adal.js

```diff
@@ -739,9 +815,7 @@ var AuthenticationContext = (function () {
     };
 
     /**
-     * Saves token from hash that is received from redirect.
-     * @param {string} hash  -  Hash passed from redirect page
-     * @returns {string} error message related to login
+     * Saves token from hash that is received from the redirect response from AAD.
```

Comment:
saves token or error in the cache received in the response from AAD. In case of id_token, it also creates the user object.